src/main/client/src/components/app/portal/portal-window/PortalWindowContainer.js

```diff
@@ -2,10 +2,16 @@
 import { connect } from 'react-redux';
 import PortalWindow from './PortalWindow';
 import { togglePortal } from '../../../../reducers/portal/actions';
+import store from '../../../../store';
 
-const mapStateToProps = state => ({
-    portalIsOpen: state.portal.togglePortal,
-});
+const mapStateToProps = (state) => {
+    const storeActiveTable = store.getState().table.features.activeTable;
+
+    return {
+        portalIsOpen: state.portal.togglePortal,
+        activeTable: storeActiveTable,
+    };
+};
```

Comment:
Storea ei tarvitse importoida, voi yksinkertaistaa:

```suggestion
const mapStateToProps = state => ({
    portalIsOpen: state.portal.togglePortal,
    activeTable: state.table.features.activeTable,
});
```

Reply:
Resolved